cucumber-messages/messages.proto

```diff
@@ -456,34 +459,39 @@ message TestStepMatched {
 }
 
 message TestStepStarted {
-  string pickleId = 1;
+  reserved 1;
   uint32 index = 2;
   Timestamp timestamp = 3;
+  string testCaseId = 4;
 }
 
 message TestStepFinished {
-  string pickleId = 1;
+  reserved 1;
   uint32 index = 2;
   TestResult testResult = 3;
   Timestamp timestamp = 4;
+  string testCaseId = 5;
 }
 
 message TestHookStarted {
-  string pickleId = 1;
+  reserved 1;
   Timestamp timestamp = 2;
+  string testCaseId = 5;
 }
 
 message TestHookFinished {
-  string pickleId = 1;
+  reserved 1;
   TestResult testResult = 2;
   Timestamp timestamp = 3;
+  string testCaseId = 5;
 }
 
 message TestResult {
   Status status = 1;
   string message = 2;
   reserved 3;
   Duration duration = 4;
+  bool will_be_retried = 5;
```

Comment:
Let's use camelCase consistently, and not snake_case.

Reply:
Fixed